app/main.py

```diff
@@ -1,6 +1,40 @@
+import os
+
+import requests
+from dotenv import load_dotenv
+from requests import Response
+
+load_dotenv()
+
+URL = "http://api.weatherapi.com/v1/current.json"
+API_KEY = os.getenv("API_KEY")
+LOCATION = os.getenv("LOCATION")
+
+
+def view_weather(response: Response) -> None:
+    weather_data = response.json()
+    print(f"Perform request to Weather API for city {LOCATION}...")
+    print(
+        f"{weather_data["location"]["country"]}/"
+        f"{weather_data["location"]["name"]} "
+        f"{weather_data["location"]["localtime"]} "
+        f"Weather: {weather_data["current"]["temp_c"]}"
+        f" Celsius {weather_data["current"]["condition"]["text"]}"
```

Comment:
The string formatting within the `print` function is incorrect. You should use single quotes for the string literals or escape the double quotes inside the f-string to avoid syntax errors. For example: `f"{weather_data['location']['country']}/"`.